docs/attributes-registry/system.md

```diff
@@ -27,157 +27,157 @@ Describes System attributes
 
 Describes System CPU attributes
 
-| Attribute                   | Type   | Description                     | Examples                 | Stability                                                        |
-| --------------------------- | ------ | ------------------------------- | ------------------------ | ---------------------------------------------------------------- |
-| `system.cpu.logical_number` | int    | The logical CPU number [0..n-1] | `1`                      | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `system.cpu.state`          | string | The state of the CPU            | `user`; `system`; `nice` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Attribute                   | Type   | Description                     | Examples            | Stability                                                        |
+| --------------------------- | ------ | ------------------------------- | ------------------- | ---------------------------------------------------------------- |
+| `system.cpu.logical_number` | int    | The logical CPU number [0..n-1] | `1`                 | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `system.cpu.state`          | string | The state of the CPU            | `idle`; `interrupt` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Are the examples added automatically now? I don't have any strong preference here but it might be that specific examples are more representative in specific cases.

Reply:
Examples come from either the `examples` markdown configuration *or* they automatically pull from the possible enum values.

See: `examples` macro in [`attribute_macros.j2`](https://github.com/open-telemetry/semantic-conventions/pull/1000/files#diff-c52795d49644cadaed9650956ba945ddb282aa8709404236217b039e7e8aa439R20)